schema/table.go

```diff
@@ -65,6 +65,33 @@ var (
 	reValidColumnName = regexp.MustCompile(`^[a-z_][a-z\d_]*$`)
 )
 
+type TableSet struct {
+	tables map[string]bool
```

Comment:
why bool, use `struct{}`? bool looks simpler but it's confusing if used for a set type.